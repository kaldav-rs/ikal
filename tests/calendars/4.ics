BEGIN:VCALENDAR
BEGIN:VTIMEZONE
TZID:Europe/Paris
X-LIC-LOCATION:Europe/Paris
BEGIN:DAYLIGHT
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
DTSTART:19700329T020000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=3
END:DAYLIGHT
BEGIN:STANDARD
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
DTSTART:19701025T030000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=10
END:STANDARD
END:VTIMEZONE
BEGIN:VEVENT
CREATED:20130503T053745Z
LAST-MODIFIED:20140709T182046Z
DTSTAMP:20140709T182046Z
UID:1686cebb-a816-4657-8d5e-3b2ce0c0589c
SUMMARY:Assemblée Générale Ordinaire 2013
DTSTART;TZID=Europe/Paris:20130516T193000
DTEND;TZID=Europe/Paris:20130516T203000
DESCRIPTION:Bonjour\,\n\nAprès un peu de retard\, nous sommes prêts pour
  organiser l\'Assemblée\nGénérale Ordinaire.\n\nEn tant que membre de l\'
 association Faimaison\, vous êtes convoqué à\nl\'assemblée Générale O
 rdinaire.\n\nCelle-ci aura lieu le 16 Mai à B17 (17\, rue Paul Bellamy)
 à 19h30.\n\nComme expliqué dans les derniers échanges\, il y aura un re
 nouvellement\nde bureau avec changement de président.\n\nL\'ordre du jour
 sera donc le suivant :\n\n- Bilan Moral\n- Bilan Financier\n- Renouvelleme
 nt du bureau.\n- Projets en cours\n\nSi vous ne pouvez pas participer vous
  pouvez donner procuration à un\nmembre de votre choix.\nPour cela\, envo
 yez un mail à bureau@faimaison.net en indiquant la personne\nmandatée.\n
 \nÀ très bientot ! \n\n-- \nGuillaume Douté
CLASS:PUBLIC
TRANSP:OPAQUE
END:VEVENT
END:VCALENDAR
