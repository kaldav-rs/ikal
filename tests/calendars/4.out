Ok(
    VCalendar {
        prodid: Text {
            params: {},
            text: "-//Nextcloud calendar v1.5.0",
        },
        version: Text {
            params: {},
            text: "2.0",
        },
        calscale: None,
        method: None,
        alarms: [],
        events: [
            VEvent {
                dtstamp: Local(
                    2014-07-09T20:20:46+02:00,
                ),
                uid: Text {
                    params: {},
                    text: "1686cebb-a816-4657-8d5e-3b2ce0c0589c",
                },
                dtstart: DateTime(
                    Naive(
                        2013-05-16T19:30:00,
                    ),
                ),
                class: Some(
                    Public,
                ),
                created: Some(
                    Local(
                        2013-05-03T07:37:45+02:00,
                    ),
                ),
                description: Some(
                    Text {
                        params: {},
                        text: "Bonjour\\,\\n\\nAprès un peu de retard\\, nous sommes prêts pour organiser l\\'Assemblée\\nGénérale Ordinaire.\\n\\nEn tant que membre de l\\'association Faimaison\\, vous êtes convoqué à\\nl\\'assemblée Générale Ordinaire.\\n\\nCelle-ci aura lieu le 16 Mai à B17 (17\\, rue Paul Bellamy)à 19h30.\\n\\nComme expliqué dans les derniers échanges\\, il y aura un renouvellement\\nde bureau avec changement de président.\\n\\nL\\'ordre du joursera donc le suivant :\\n\\n- Bilan Moral\\n- Bilan Financier\\n- Renouvellement du bureau.\\n- Projets en cours\\n\\nSi vous ne pouvez pas participer vous pouvez donner procuration à un\\nmembre de votre choix.\\nPour cela\\, envoyez un mail à bureau@faimaison.net en indiquant la personne\\nmandatée.\\n\\nÀ très bientot ! \\n\\n-- \\nGuillaume Douté",
                    },
                ),
                geo: None,
                last_modified: Some(
                    Local(
                        2014-07-09T20:20:46+02:00,
                    ),
                ),
                location: None,
                organizer: None,
                priority: None,
                sequence: None,
                status: None,
                summary: Some(
                    Text {
                        params: {},
                        text: "Assemblée Générale Ordinaire 2013",
                    },
                ),
                transp: Some(
                    Opaque,
                ),
                url: None,
                recurid: None,
                rrule: None,
                dtend: Some(
                    DateTime(
                        Naive(
                            2013-05-16T20:30:00,
                        ),
                    ),
                ),
                duration: None,
                attach: [],
                attendee: [],
                categories: [],
                comment: [],
                contact: [],
                exdate: [],
                rstatus: [],
                related_to: [],
                resources: [],
                rdate: [],
                alarms: [],
                x_prop: {},
                iana_prop: {},
            },
        ],
        freebusy: [],
        journals: [],
        todo: [],
        timezones: [
            VTimezone {
                tzid: Text {
                    params: {},
                    text: "Europe/Paris",
                },
                last_modified: None,
                tzurl: None,
                standard: [
                    Standard {
                        dtstart: DateTime(
                            Naive(
                                1970-10-25T03:00:00,
                            ),
                        ),
                        tzoffsetto: +01:00,
                        tzoffsetfrom: +02:00,
                        rrule: Some(
                            Recur {
                                freq: Yearly,
                                until: None,
                                count: None,
                                interval: 1,
                                by_second: [],
                                by_minute: [],
                                by_hour: [],
                                by_day: [
                                    WeekdayNum {
                                        weekday: Sunday,
                                        ord: Some(
                                            -1,
                                        ),
                                    },
                                ],
                                by_monthday: [],
                                by_yearday: [],
                                by_weekno: [],
                                by_month: [
                                    10,
                                ],
                                by_setpos: [],
                                wkst: None,
                            },
                        ),
                        comment: [],
                        rdate: [],
                        tzname: [
                            Text {
                                params: {},
                                text: "CET",
                            },
                        ],
                        x_prop: {},
                        iana_prop: {},
                    },
                ],
                daylight: [
                    Daylight {
                        dtstart: DateTime(
                            Naive(
                                1970-03-29T02:00:00,
                            ),
                        ),
                        tzoffsetto: +02:00,
                        tzoffsetfrom: +01:00,
                        rrule: Some(
                            Recur {
                                freq: Yearly,
                                until: None,
                                count: None,
                                interval: 1,
                                by_second: [],
                                by_minute: [],
                                by_hour: [],
                                by_day: [
                                    WeekdayNum {
                                        weekday: Sunday,
                                        ord: Some(
                                            -1,
                                        ),
                                    },
                                ],
                                by_monthday: [],
                                by_yearday: [],
                                by_weekno: [],
                                by_month: [
                                    3,
                                ],
                                by_setpos: [],
                                wkst: None,
                            },
                        ),
                        comment: [],
                        rdate: [],
                        tzname: [
                            Text {
                                params: {},
                                text: "CEST",
                            },
                        ],
                        x_prop: {},
                        iana_prop: {},
                    },
                ],
                x_prop: {
                    "X-LIC-LOCATION": ContentLine {
                        params: {},
                        value: "Europe/Paris",
                    },
                },
                iana_prop: {},
            },
        ],
        x_prop: {},
        iana_prop: {},
    },
)
